app/consumer-democracy/app.go

```diff
@@ -468,6 +522,23 @@ func New(
 
 	app.EvidenceKeeper = *evidenceKeeper
 
+	adminRouter := ccvgovtypes.NewRouter()
+	adminRouter.AddRoute(ccvgovtypes.RouterKey, ccvgovtypes.ProposalHandler).
+		AddRoute(paramproposal.RouterKey, params.NewParamChangeProposalHandler(app.ParamsKeeper)).
+		AddRoute(upgradetypes.RouterKey, upgrade.NewSoftwareUpgradeProposalHandler(app.UpgradeKeeper))
+
+	app.AdminmoduleKeeper = *adminmodulemodulekeeper.NewKeeper(
+		appCodec,
+		keys[adminmodulemoduletypes.StoreKey],
+		keys[adminmodulemoduletypes.MemStoreKey],
+		adminRouter,
+		// this allows any type of proposal to be submitted to the admin module (everything is whitelisted)
+		// projects will implement their functions to define what is allowed for provider and consumer admins
+		func(ccvgovtypes.Content) bool { return true },
+		func(ccvgovtypes.Content) bool { return true },
```

Comment:
this is nice. Maybe an example of a switch statement with various proposal types to show how you'd include or exclude specific ones would be nice to have?

Reply:
Yes, we already have similar function for democracy consumer chain that implements whitelisting based on proposal type. We will change this comment to make reference to that function.